tests/opwen_email_server/mailers/test_wikipedia.py

```diff
@@ -0,0 +1,41 @@
+from unittest import TestCase
+
+import responses
+
+from opwen_email_server.mailers.wikipedia import WikipediaEmailFormatter
+
+
+class WikipediaServiceTests(TestCase):
+    def setUp(self):
+        self._formatter = WikipediaEmailFormatter()
+
+    def test_no_results(self):
```

Comment:
I would suggest for the failure test cases to also set up mock responses (like the success test case) for two reasons: (1) the tests won't need internet access to run (2) the tests will be robust to Wikipedia changing in the future.